### PR TITLE
Add GitHub issue template

### PR DESCRIPTION
**Pull request by thagikura**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- [ ] I have searched [existing issues](https://github.com/google/flexbox-layout/issues) and confirmed this is not a duplicate
+
+## Issues and steps to reproduce
+*Please replaces this with steps to reproduce your issue.*
```
Comment (yaraki) [inline .github/issue_template.md]: s/replaces/replace/

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Expected behavior
+*Please describe what you expected would happen.*
+
+## Version of the flexbox library
+*e.g. 0.2.6, 0.3.0-alpha3*
+
+## Link to code
+*Please link to the code we can use to reproduce this issue.*
+*A complete project we can build/run is more preferred, if not, please show*
```
Comment (yaraki) [inline .github/issue_template.md]: s/more preferred, if not,/preferred. If you can't provide one,/

File: .github/issue_template.md (hunk 1 above)
Comment (thagikura) [inline .github/issue_template.md]: Done.

File: .github/issue_template.md (hunk 2 above)
Comment (thagikura) [inline .github/issue_template.md]: Done

**Pull request merged**